rust/tools/authorization-logic/src/parsing/AuthLogic.g4

```diff
@@ -139,7 +144,8 @@ ATTRIBUTE: 'attribute';
 
 // Identifiers wrapped in quotes are constants whereas
 // identifiers without quotes are variables.
-ID : ('"')? [_a-zA-Z][_a-zA-Z0-9/.#:]* ('"')?;
+VARIABLE : [_a-zA-Z][_a-zA-Z0-9/.#:]*; 
+CONSTANT : '"' [\n _a-zA-Z+-][\n _a-zA-Z0-9/.#:+=-]* '"';
```

Comment:
@bgogul I want to specifically point out newlines are now allowed in constants since this could be controversial (but I think it's OK). See the hyphenIDs file for the motivation for this which is to allow the contents of public key files as constants

Reply:
Should we use a different delimiters for multiline strings? e.g., C++ raw literals are delimited by `R"(` and `)"` and in Kotlin, we use `"""` for multiline strings.